packages/wonder-blocks-dropdown/components/action-item.js

```diff
@@ -80,6 +81,8 @@ export default class ActionItem extends React.Component<ActionProps> {
         indent: false,
     };
 
+    static contextTypes = {router: PropTypes.any};
```

Comment:
Thanks for fixing this.  What's interesting is that the clientNav tests were passing for all components not just Button.  I think the issue is https://github.com/Khan/wonder-blocks/blob/master/packages/wonder-blocks-core/util/get-clickable-behavior.js#L37-L39.  I think we should make `clientNav` mandatory when using the router.  Otherwise it's hard to know whether you'll get client-side navigation just by looking at a call-site whether you'll get client-side navigation or not.  🛑